sdk/batch/Microsoft.Azure.Batch/src/AddTasksWorkflowManager.cs

```diff
@@ -92,7 +94,7 @@ public async System.Threading.Tasks.Task AddTasksAsync(IEnumerable<CloudTask> ta
 
             if (original != HasNotRun)
             {
-                throw new RunOnceException(string.Format(BatchErrorMessages.CanOnlyBeRunOnceFailure, this.GetType().Name));
+                throw new RunOnceException(string.Format(CultureInfo.InvariantCulture, BatchErrorMessages.CanOnlyBeRunOnceFailure, this.GetType().Name));
```

Comment:
How are you determining Invariant v Current? System.Globalization.CultureInfo.CurrentCulture

Reply:
I made all of our predefined exception types invariant culture currently because AFAIK localization isn't performed on these SDKs, and mostly printing things like type names (as is the case here) should be `Invariant` anyway I think